beesdoo_shift_attendance/models/attendance_sheet.py

```diff
@@ -33,6 +33,14 @@ def pre_filled_task_type_id(self):
         task_types = self.env["beesdoo.shift.type"]
         return task_types.browse(tasktype_id)
 
+    @api.model
+    def _default_attendance_sheet_shift_sheet(self):
+        parameters = self.env["ir.config_parameter"].sudo()
+        return parameters.get_param(
+            "beesdoo_shift_attendance.attendance_sheet_default_shift_state",
+            default="absent_2",
```

Comment:
Ought this default not be set in res_config_settings?

Reply:
The default argument you mean ?

`res_config_settings` is a special wizard model that is used to write in `ir.config_parameter` (usually). The default on `res_config_settings` only therefore add a default value on the wizard when it is opened for the first time. It is not a system-wide default.